### PR TITLE
feat(authoring): SD-3698 Improved usability of Subject and Category popup

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
SD-3698

### CODE REVIEW

File: client/app/scripts/superdesk/ui/ui.js [hunk 1]
```diff
@@ -397,6 +397,69 @@ define([
...
+
+                            if (prevElem.length > 0) {
+                                prevElem.focus();
+                            } else {
+                                inputField.focus();
+                                categoryButton.focus();
+                            }
+                        }, keyboardOptions);
+
+                        keyboardManager.push('down', function () {
```
Comment (petrjasek) [inline client/app/scripts/superdesk/ui/ui.js]: I think it will also need to pop it at some point, otherwise this will leak memory

Comment (petrjasek) [inline client/app/scripts/superdesk/ui/ui.js]: like on dropdown close

**Comment by ioanpocol** [comment]
:+1:

**Comment by Aca-jov** [comment]
:+1:

**Pull request merged**